modules/op-restarting-or-canceling-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -3,13 +3,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="restarting-or-canceling-pipeline-run-using-pipelines-as-code_{context}"]
-= Restarting or canceling a pipeline run using {pac}
+= Manually starting, restarting or canceling a pipeline run using {pac}
 
-You can restart or cancel a pipeline run with no events, such as sending a new commit to your branch or raising a pull request. To restart all pipeline runs, use the *Re-run all checks* feature in the GitHub App.
+You can manually start, restart, or cancel a pipeline run with no events, such as sending a new commit to your branch or raising a pull request. To restart all pipeline runs, use the *Re-run all checks* feature in the GitHub App.
 
-To restart all or specific pipeline runs, use the following comments:
+To manually start or restart all or specific pipeline runs, use the following comments:
 
-* The `/test` and `/retest` comment restarts all pipeline runs.
+* The `/test` and `/retest` comment restarts all pipeline runs that match events for this pipeline run.
 
 * The `/test <pipeline_run_name>` and `/retest <pipeline_run_name>` comment starts or restarts a specific pipeline run. You can use this command to start any {pac} pipeline run on the repository, whether or not it was triggered by an event for this pipeline run.
```

Comment:
maybe we need to mention that the specific pipelinerun is a pipelinerun with a name that is located in the .tekton directory (or subdirectories)